src/constants.ts

```diff
@@ -8,7 +8,7 @@ export const PROVISIONING_API = '/api/provisioning/v1';
 export const COMPLIANCE_API = '/api/compliance/v2';
 export const CREATE_BLUEPRINT = `${IMAGE_BUILDER_API}/blueprints`;
 export const EDIT_BLUEPRINT = `${IMAGE_BUILDER_API}/blueprints`;
-
+export const isOnPremise = process.env.IS_ON_PREMISE;
```

Comment:
I wonder if we need this? It doesn't seem like a constant to me, since it can be either true or false. :thinking: